### PR TITLE
september-change-logs

### PR DESCRIPTION
**Pull request by kghamilton89**
## September Change Logs

* Source Document: `base-url` + `/issues?query_id=19`

* Removed entries:
  * `5684`
  * `5704`
  * `5656`

* Other issues:
  * `5672` and `5675` are identical issues. Unclear whether to remove one or the other.
  * `5688` has no **Category** entry and it is unclear which one to use.
  * `5668`: Unclear which documentation link is needed. Currently, using Rule Engine docs.
  * `5653`: Unclear how to define issue specifically.

### CODE REVIEW

**Comment by rodionos** [comment]
I removed and re-classified some issues, please re-insert bug/feature counters.

**Comment by kghamilton89** [comment]
Bugs: `31`
Features: `20`

**Comment by rodionos** [comment]
@kghamilton89 - please incorporate changes from https://github.com/axibase/axibase-charts-vscode/blob/master/CHANGELOG.md, under the Charts section

**Comment by kghamilton89** [comment]
@rodionos 
* Changes from source document added to Charts section.
* I began numbering from `0001`.
* I did my best to properly classify each issue as either a bug or feature and apply title rules accordingly.

File: changelogs/2018_sep/README.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+5503|UI|Feature|UI: Add [Change Log](../README.md) link to **Support** tab in top menu.
+5231|core|Feature|[Replacement Tables](../../sql/examples/lookup.md#replacement-tables): Add **Type** and **Information** fields.
+5097|sql|Bug|[SQL](../../sql/README.md): Allow expressions in **Entity** column to be exposed to parent query.
+|4945|rule engine|Bug|[Rule Engine](../../rule-engine/README.md): Erroneous number formatting.
+
+## Charts
+
+**Issue**| **Category**    | **Type**    | **Subject**
+-----|-------------|---------|----------------------
+|0001|plugin| Feature | Remove suggestions based on [Levenshtein distance](https://en.wikipedia.org/wiki/Levenshtein_distance) due to low match quality.
```
Comment (rodionos) [inline changelogs/2018_sep/README.md]: Lets assign numbers in reverse order, the smallest at the bottom, this would be similar to other sections

Comment (kghamilton89) [reply changelogs/2018_sep/README.md]: Issue order revered by `reverse-issue-order` commit.

**Pull request merged**